apis/kubeeye/v1alpha1/clusterinsight_types.go

```diff
@@ -31,6 +31,22 @@ type ClusterInsightSpec struct {
 	AuditPeriod string `json:"auditPeriod"`
 }
 
+type Plugins struct {
```

Comment:
These structs are useless. I think these can be deleted.

Reply:
I think we should add a `Plugins` field to the `ClusterInsightSpec` struct.

such as：
```
type ClusterInsightSpec struct {
	AuditPeriod string   `json:"auditPeriod"`
        Plugins     []string `json:"plugins"`
}
```